todo_app/e2e_test.py

```diff
@@ -30,13 +58,12 @@ def app_with_temp_board():
     yield application
 # Tear Down
     thread.join(1)
-    #delete_trello_board(board_id)
+    delete_trello_board(board_id)
 
 @pytest.fixture(scope="module")
 def driver():
     #with webdriver.Firefox() as driver
     driver = webdriver.Firefox()
-    driver.get("http://localhost:5000/")
     yield driver
```

Comment:
The Firefox windows don't close after the tests because we haven't closed the driver, we should either do that manually with:
```python
    driver = webdriver.Firefox()
    yield driver
    driver.close()
```
or we can use a with block to handle that for us:
```python
    with webdriver.Firefox() as driver:
        yield driver
```